deepform/data/add_features.py

```diff
@@ -137,7 +146,7 @@ def label_tokens(tokens, labels, max_token_count):
     for col_name, label_value in labels.items():
         tokens[col_name] = 0.0
         match_fn = LABEL_COLS[col_name]
-
+        max_token_count = MAX_TOKENS_BY_TARGET[col_name]
```

Comment:
This is a good use case for `get`, so that if we ever add fields and don't update the dictionary we don't crash the program with a `KeyError` exception. We can optionally also leave targets other than advertiser undefined and just use the default of 3.
```suggestion
        max_token_count = MAX_TOKENS_BY_TARGET.get(col_name, default=3)
```